src/macrofier/types.mjs

```diff
@@ -120,17 +120,18 @@ const insertConstMacros = (content, schema, module, name) => {
   return content
 }
 
-
-
-const insertEnumMacros = (content, schema, module, name) => {
+const insertEnumMacros = (content, schema, module, name, suffix) => {
   const template = content.split('\n')
 
   for (var i = 0; i < template.length; i++) {
     if (template[i].indexOf('${key}') >= 0) {
-      template[i] = schema.enum.map(value => {
+      template[i] = schema.enum.filter(value => value).map(value => {
         return template[i].replace(/\$\{key\}/g, safeName(value))
           .replace(/\$\{value\}/g, value)
       }).join('\n')
+      if (suffix !== ".cpp") {
```

Comment:
Would be better to do this with a feature flag in sdk.config.json

Reply:
ok

Reply:
updated config for this

Reply:
removed enumSufix from the config and used delimiter macro to handle this